android/app/build.gradle

```diff
@@ -101,9 +101,12 @@ android {
         ndk {
             abiFilters "armeabi-v7a", "x86"
         }
-        manifestPlaceholders = [manifestApplicationId: "${applicationId}",
-                        onesignal_app_id: "aa46a500-ab1c-4127-b9ff-e7373da3ce35",
-                        onesignal_google_project_number: "185558680648"]
+        manifestPlaceholders = [
+            manifestApplicationId: "${applicationId}",
+            onesignal_app_id: "aa46a500-ab1c-4127-b9ff-e7373da3ce35",
+            onesignal_google_project_number: "185558680648",
+            gmaps_key: System.getenv("GOOGLE_MAPS_API_KEY") ?: "",
```

Comment:
@drewvolz So here, you use the env variable `GOOGLE_MAPS_API_KEY`, but in the `scripts/travis/script.sh` file you echo the `GMAPS_KEY` env variable.

According to Travis, the correct name is `GMAPS_KEY`.

I notice that in https://github.com/StoDevX/AAO-React-Native/pull/897/commits/f8b3637c204da811ddc01a57f12de52bf46a5f6d, you changed from `GMAPS…` to `GOOGLE_MAPS…` — are you sure that that's correct on Travis?